src/Modules/Payments/Tests/IntegrationTests/SeedWork/TestBase.cs

```diff
@@ -96,9 +96,9 @@ protected async Task<T> GetLastOutboxMessage<T>()
             }
         }
 
-        protected static void AssertEventually(IProbe probe, int timeout)
+        protected static async Task AssertEventually(IProbe probe, int timeout)
```

Comment:
@kgrzybek what do you think about Async postfix? I didn't add here because in the file there is async method GetEventually without the postfix.

Reply:
We can leave it without postfix.